src/main/java/de/tum/in/www1/artemis/service/dto/athena/TextFeedbackDTO.java

```diff
@@ -2,12 +2,15 @@
 
 import jakarta.validation.constraints.NotNull;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+
 import de.tum.in.www1.artemis.domain.Feedback;
 import de.tum.in.www1.artemis.domain.TextBlock;
 
 /**
  * A DTO representing a Feedback on a TextExercise, for transferring data to Athena and receiving suggestions from Athena
  */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
 public record TextFeedbackDTO(long id, long exerciseId, long submissionId, String title, String description, double credits, Long structuredGradingInstructionId,
         Integer indexStart, Integer indexEnd) implements FeedbackDTO {
```

Comment:
The AthenaFeedbackSendingServiceTest is failing.

Setting

```
        textFeedback.setText("some text");
        textFeedback.setDetailText("some detail text");
```

in the setup method of that class resolved this problem for me. Can you have a look? I'm not sure if these values are supposed to be filled - if not, you'll have to adjust the checks.